site/content/entry/citra-progress-report-2019-q1.md

```diff
@@ -0,0 +1,295 @@
+He based his implementation on the old PR, but vastly improved it by rewriting many parts and fixing the many design flaws it had.
+Thanks to another developer [zhaowenlan1779](https://github.com/zhaowenlan1779), who implemented the [UI](https://github.com/citra-emu/citra/pull/4610) for cheats, the full cheat code support is now merged into Nightly and Canary.
+
+## #5 Custom Texture Support
+
+As you've seen thus far, we've had our hands full between Pokémon X/Y, cheats, controller rewrite, and more.
+While we still want this feature, it's currently not in development, and we don't have any estimate for when someone will add it either.
+Stay tuned, as the open nature of the project means that this can change at a moment's notice.
+You never know when a current Citra developer might pick it up, or when a new contributor might submit the code for review!
+
+## #6 [Amiibo Support](https://github.com/citra-emu/citra/pull/4337) by [FearlessTobi](https://github.com/FearlessTobi)
+
+Developer [FearlessTobi](https://github.com/FearlessTobi) works tirelessly to try and keep the code that's shared between the sister project [yuzu](https://github.com/yuzu-emu/yuzu) up-to-date with Citra.
+Recently, a yuzu developer [ogniK](https://github.com/ogniK5377) added support for Amiibo to yuzu, and Tobi really wanted to know how much could just be copy pasted into Citra as well.
+
+It turns out, a fair bit, but not quite enough to get it working out of the box.
+With some extra research and some help from [ogniK](https://github.com/ogniK5377), Tobi was able to work out the differences and bring Amiibo support to Citra in October.
+This works by using `Virtual Amiibo` files, which emulates the scanning of them, and to obtain these `Virtual Amiibo` files from your physical amiibo, you can either use [TagMo](https://github.com/HiddenRamblings/TagMo/releases) app for android or a [3DS homebrew](https://github.com/moriczgergo/amibac/releases/).
+
+{{< sidebyside "image" "/images/entry/citra-progress-report-2019-q1/"
+    "amiibo.png"
+    "amiibo2.png" >}}
+
+***Note: The Amiibo implementation isn't fully complete and needs a bit of work to fix outstanding issues.***
+
+## #7 [Microphone Support](https://github.com/citra-emu/citra/pull/4671) by [jroweboy](https://github.com/jroweboy)
+
+Developer [jroweboy](https://github.com/jroweboy) picked up this task back in November, but ran into a roadblock just as he was about to finish it.
+There was some bug in the code, due to which games just weren't accepting the mic input, but after reviewing it several times and debugging it, he just couldn't figure out why.
+
+In the past week, [FearlessTobi](https://github.com/FearlessTobi) took an interest in the code for mic, and decided to try and clean it up a bit.
+During his clean-up, he found a bug or rather a typo in the code because of which the feature wasn't working.
```

Comment:
it wasn't a typo. it was just a really dumb mistake that i overlooked. i mixed up bits and bytes when working with the sizeof the mic data, so it wouldn't copy enough data from the mic